src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/xlf/BuildPropertyPage.VisualBasic.xaml.cs.xlf

```diff
@@ -304,7 +304,7 @@
       </trans-unit>
       <trans-unit id="EnumValue|OptionCompare.Binary|DisplayName">
         <source>Binary</source>
-        <target state="translated">Binární</target>
+        <target state="translated">Binary</target>

```

Comment:
Did you change these manually?

Reply:
Yes, I did. The string replacements do not need translation

Reply:
This is overridden, changing values manually is not how we are supposed to edit these values.